app/views/media/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<h2>List of Bros</h2>
+
+  <section class="media-table">
+    <h4>Sound Bros</h4>
+    <%= render partial: "media_table", locals: { collection: order_by_votes(@albums) } %>
+  </section>
+  <section class="media-table">
+    <h4>Word Bros</h4>
+    <%= render partial: "media_table", locals: { collection: order_by_votes(@books) } %>
+  </section>
+  <section class="media-table">
+    <h4>Visual Bros</h4>
+    <%= render partial: "media_table", locals: { collection: order_by_votes(@movies) } %>
+  </section>
```

Comment:
This view is already very clean code, as you made a good choice to pull out the media_table, but you could definitely find a way to push this so it's even shorter and cleaner ;)